src/tools/perf/perftest_context.h

```diff
@@ -69,15 +70,36 @@ perftest_params_release_msg_size_list(perftest_params_t *params)
 }
 
 
+static inline ucs_status_t
+perftest_params_check_api(ucx_perf_api_t dst_api, ucx_perf_api_t src_api)
+{
+    if (src_api == UCX_PERF_API_LAST) {
+        ucs_error(PARAMS_MERGE_PREFIX": source API must be defined");
+        return UCS_ERR_INVALID_PARAM;
+    }
+
+    if ((dst_api != src_api) ||
```

Comment:
i might be confused but if server options are only `-K mlx5_0:1`, dst_api is UCX_PERF_API_LAST, but we still fail? If yes I think the check should rather be:
```
if ((dst_api != UCX_PERF_API_LAST) && (dst_api != src_api)) {
```